onnxruntime/core/providers/cuda/reduction/reduction_functions.cu

```diff
@@ -396,6 +396,8 @@ template void reduce_matrix_rows<float, float>(
   const float* data, float* output, int m, int n);
 template void reduce_matrix_rows<double, double>(
   const double* data, double* output, int m, int n);
+template void reduce_matrix_rows<int64_t, int64_t>(
+  const double* data, double* output, int m, int n);
```

Comment:
double or int64 ?

Reply:
Probably both data and output are int64 (going based on the pattern above)

Reply:
Yes, got it. Looks after fixing this, the exception was:
 Error : no instance of function template "onnxruntime::cuda::reduce_matrix_rows" matches the specified type

I have changed it to CPU only for now.